nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -301,9 +328,8 @@ def _apply(self,
             node = nncf_graph.get_node_by_name(target_node_name)
             if quantization_target_point.type == TargetType.OPERATION_WITH_WEIGHTS:
                 try:
-                    input_tensor_names, _ = self._backend_entity.get_tensor_names(node)
-                    weight_initializer_name = input_tensor_names[1]
-                    weight_tensor = self._backend_entity.get_initializer_value(model, weight_initializer_name)
+                    weight_initializer_name, weight_tensor = self._backend_entity.get_weight_tensor(model,
+                                                                                                    node.node_name)
                     # If the nodes share one weight tensor, we should have only one quantizer on that
                     if weight_initializer_name in weight_initializer_names:
                         continue
```

Comment:
Is it possible to use `ONNXGraph.is_node_shared(onnx_node)` method instead of `weight_initializer_names` to create a `quantizer_insertion_command` once for shared nodes?